content/guides/developer-guides/multi-cloud/_index.md

```diff
@@ -61,7 +61,7 @@ docker-compose version 1.22.0, build 1719ceb
 ### Install and start SODA Multi-cloud Server
 Download the source code:
 ```bash
-mkdir -p /root/gopath/src/github.com/sodafoundation/
+mkdir cd -p /root/gopath/src/github.com/sodafoundation/
```

Comment:
What does this command do? We already use mkdir with the -p option to create the directory structure.
The correct way would be to provide the path in the clone command as :  
`git clone https://github.com/sodafoundation/multi-cloud.git /root/gopath/src/github.com/sodafoundation/`

or  

Change the order of the commands. Navigate to the `/root/gopath/src/github.com/sodafoundation/` folder first and then clone.